aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -0,0 +1,235 @@
+    ```
+        \dt
+    ``` 
+    We can now manipulate the database by running the appropriate sql commands under this postgres client.
+
+- Exit the postgres client using
+    ```
+        \q
+    ```
+
+- Create a .env file which must be of type **All Files** 
+  * Note that you are in the directory vms\vms (you have to create the .env file here)
+  * Just copy the contents of .env Example File to your created file
+  * Save it.
+    
+### Setting up the database
+
+- Create migrations for the database
+    ```bash
+      python manage.py makemigrations auth volunteer administrator organization event job shift registration
+    ```
+    
+- Apply migrations to database
+    ```bash
+      python manage.py migrate --noinput --traceback --settings=vms.settings
+    ```
+    
+- Populate the database for django-cities-light
+    ```bash
+      python manage.py cities_light
+    ```
+    
+- After running the command
+    ```bash
+      python manage.py runserver
+    ```
+    
+- Check that the project is running correctly by browsing to
+    ```
+      http://127.0.0.1:8000
+    ```
+  Note: The default web browser is Firefox.
+    
+### Steps to run tests
+
+- Automated tests in VMS require you to setup geckodriver in your path. For that, download geckodriver v0.20.0
+  [Geckodriver v0.20.0](https://github.com/mozilla/geckodriver/releases)
+  * Move to Geckodriver v0.20.0
+  * Then to Assets
+  * Download **geckodriver-v0.20.0-win64.zip** or **geckodriver-v0.20.0-win32.zip** depending on your system
+  * After downloading, copy & paste the downloaded geckodriver folder in the venv folder & add to path.
+
+- To execute all tests VMS, use this command
+    ```bash
+      python manage.py test -v 2
+    ```
+
+- To execute tests in a particular file use
+    ```bash
+      python manage.py test <app_name> -v 2 
+    ```
+
+- To execute a test file inside an app use
+    ```bash
+      python manage.py test <app_name>.tests.<test_file_name> -v 2
+    ```
+
+- If all tests pass, `OK` will be received at the end.
+
+- For automated tests, if any of the tests fail its not necessary that there is something wrong. To confirm if the the test is actually wrong you have to test it in headless mode.
+
+    Note: For automated testing, currently VMS uses the Firefox version 60, selenium version 3.4.0 and geckodriver version 0.20.1.
+
+    Note: Some of the test may be failing due to the incorrect permission given to the media folder, If media folder (srv directory) is already created on your system , then make sure it has read, write & execute permission.
+   
+- In case you do not find the directory make one in vms\vms.
+
+- After creating the directory, make sure it has read, write & execute permission, you can check the **Properties** of the directory for checking the permission.
+
+### Stop the PostgreSQl Server
+
+- To stop the postgreSQL Server there are 3 ways
+
+## Command Prompt
+    ```bash
```

Comment:
same here, I can see the word `bash` here as well after preview which shouldn't happen ideally.